python/lsst/ts/salkafka/run_salkafka_producer.py

```diff
@@ -100,6 +101,10 @@ async def run_salkafka_producer():
     log.addHandler(logging.StreamHandler())
     log.setLevel(args.loglevel)
 
+    semaphore_file = pathlib.Path("/tmp", "RUNNING")
```

Comment:
uhm... so, what happens if you have more than one producers running at the same time? Do you want to attach the PID of the process to the file? Also, you may want to have a name like `SALKAFKA_PRODUCER_RUNNING`, no?

Reply:
Since we run the producers in containers and the write is local to the container, there is no chance of them stepping on each other. The only time that would happen is if you are running them as bare system processes or have all the container /tmp dirs mapped to the same local directory. The semaphore is only meant to be used in the k8s context. I cannot use the PID in the filename since that is random and not know at the time the readiness probe is defined on the pod. I don't mind making the name SALKAFKA_PRODUCER_RUNNING if you think that gives more clarity.